content/definition.md

```diff
@@ -8,7 +8,7 @@ Software is considered Ethical Source if it meets the following criteria:
 1. It is distributed with source code and can be freely used or combined with other software.
 1. It is developed or maintained in public view and welcoming of public contributions.
 1. Its community of contributors is governed by a code of conduct that is consistently and fairly enforced.
-1. It is distributed under a license that prohibits the software from being used any individual or organization which directly or indirectly facilitates, encourages, manipulates, coerces, or forces people to engage in behaviors that are in opposition to a specified framework of social good.
+1. It is distributed under a license that prohibits the software from being used by any individual or organization which directly or indirectly facilitates, encourages, manipulates, coerces, or forces people to engage in behaviors that are in opposition to a specified framework of social good, provided that the framework does not infringe on the rights enumerated in the United Nations Universal Declaration of Human Rights.
```

Comment:
"It is distributed under a license that prohibits its use "?

Reply:
Not sure I understand this one, please expand?

Reply:
Never mind :)